src/client/src/orangehrmAdminPlugin/pages/languagePackage/language-package.scss

```diff
@@ -27,20 +29,16 @@
 }
 
 .oxd-table-cell-actions{
-    justify-content: end;
-}
-
-.orangehrm-dialog-modal{
-    width: '90%';
-    min-width: '800px';
+    @include oxd-respond-to('xs') {
+        justify-content: end;
+    }  
+    @include oxd-respond-to('lg') {
```

Comment:
```suggestion
    @include oxd-respond-to('md') {
```

Reply:
Not work for some screen sizes. like iPadAir (820*1180)

Reply:
Done